src/Command/RandomIssueCommand.php

```diff
@@ -59,6 +56,21 @@ protected function configure(): void
             InputOption::VALUE_OPTIONAL,
             'Do you want issues for #hacktoberfest?'
         );
+
+        $this->addOption(
+            'source',
+            's',
+            InputOption::VALUE_OPTIONAL,
+            'Which source should be used? Currently github and gitlab are supported',
+            'github'
+        );
+
+        $this->addOption(
```

Comment:
Can we maybe add the PAT using a `.env` file and parse it using `symfony/dotenv`?

Reply:
I added the symfony/dotenv as dependency and load it in the `bin/rip`.

The `file_exists` is added, because all users of github only, should'nt see an error, that they need a `.env` file